vip-integrations.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+use InvalidArgumentException;
+
+defined( 'ABSPATH' ) || die();
+
+require_once __DIR__ . '/vip-integration-helper/integrations/integration.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/integrations.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/block-data-api.php';
+
+class VipIntegrations {
+	public function setup() {
+		add_action( 'muplugins_loaded', [ $this, 'register' ], /* priority */ 4 );
+		add_action( 'muplugins_loaded', [ $this, 'activate' ], /* priority */ 6 );
+		add_action( 'muplugins_loaded', [ $this, 'integrate' ], /* priority */ 8 );
+	}
+
+	// Actions
+
+	public function register(): void {
+		add_action( 'register_vip_integrations', function() {
+			// Register VIP integrations here
+			Integrations::instance()->register( 'block-data-api', BlockDataApi::class );
+		} );
```

Comment:
```suggestion
		// Register VIP integrations here
		Integrations::instance()->register( 'block-data-api', BlockDataApi::class );
```

The `add_action` seems unnecessary here?

Reply:
I made some changes to how we handle integrations by removing a lot of these extraneous filters. I'll leave a comment below with more information.